pom.xml

```diff
@@ -84,6 +84,11 @@
             <artifactId>guice</artifactId>
             <version>4.2.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.10.1</version>
```

Comment:
Pls change to latest version 
**_<dependency>
  <groupId>com.fasterxml.jackson.core</groupId>
  <artifactId>jackson-databind</artifactId>
  <version>2.10.2</version>
</dependency>_**

Reply:
fixed